lib/pmcrypto.js

```diff
@@ -1,55 +1,44 @@
-import { setInstance, setConfig } from './openpgp';
+import { setConfig } from './openpgp';
 import { encodeUtf8, decodeUtf8, decodeUtf8Base64, encodeUtf8Base64, encodeBase64, decodeBase64 } from './utils';
 
-export const init = (openpgp) => {
-    if (!openpgp) {
-        throw new Error('OpenPGP required');
-    }
-    setInstance(openpgp);
-    setConfig(openpgp);
+export const init = () => {
```

Comment:
This is going to be quite confusing with IDE suggestion, as the `openpgp` argument will be displayed in the tooltip. Plus, as soon as we upgrade to TS, compilation will fail if they pass it. And since we'll expose the worker, I'm not sure if this will be used by the apps anymore.

Reply:
We could do
```suggestion
export const init = function() {
    if (arguments.length) {
        throw new Error('Loading OpenPGP separately is no longer required');
    }
```